latest/index.bs

```diff
@@ -207,8 +207,52 @@ keys as specified below for discovering certain types of data, especially images
 
 Metadata about the multiple resolution representations of the image can be
 found under the "multiscales" key in the group-level metadata.
-The specification for the multiscale (i.e. "resolution") metadata is provided
-in [zarr-specs#50](https://github.com/zarr-developers/zarr-specs/issues/50).
+"multiscales" contains a list of dictionaries where each entry describes a multiscale image.
+
+"multiscales" MUST contain the field "datasets", which is a list of dictionaries describing
+the arrays storing the individual resolution levels. Each dictionary in "datasets" MUST
+contain the field "path", whose value contains the path to the array for this resolution relative
+to the root zarr group. The "path"s MUST be ordered from largest (i.e. highest resolution) to smallest.
+
+It SHOULD contain the field "name".
+
+It SHOULD contain the field "version", which indicates the version of the
+multiscale metadata of this image (current version is 0.1).
+
+It SHOULD contain the field "axes", which is a list of dimension names of the axes.
+If given, the values MUST be unique and one of `{"t", "c", "z", "y", "x"}`.
+The number of values MUST be the same as the number of dimensions of the arrays corresponding to this image.
+
+It SHOULD contain the field "type", which gives the type of downscaling method used to generate the multiscale image pyramid.
+It SHOULD contain the field "metadata", which contains a dictionary with additional information about the downscaling method.
+
+```json
+{
+    "multiscales": [
+        {
+            "version": "0.1",
+            "name": "example",
+            "datasets": [
+                {"path": "0"},
+                {"path": "1"},
+                {"path": "2"}
+            ],
+            "axes": [
+                "t", "c", "z", "y", "z"
+            ],
+            "type": "gaussian",
+            "metadata": {                                       # the fields in metadata depend on the downscaling implementation
+                "method": "skimage.transform.pyramid_gaussian", # here, the paramters passed to the skimage function are given
+                "version": "0.16.1",
+                "args": "[true]",
+                "kwargs": {"multichannel": true}
```

Comment:
Side note probably outside the original scope of this PR, this example was copied from https://github.com/zarr-developers/zarr-specs/issues/50 and reads very Python-specific in a specification that is expected to be increasingly language-agnostic.

Reply:
Yes, I was also wondering about this when I copied from the example.

Reply:
I'd leave the python example _somewhere_, but we can also add a Java one, e.g. from bioformats2raw.

Reply:
Ok, we can also leave it as it is; I don't have any strong feelings either way here.